src/ConstantsMixins.scss

```diff
@@ -1,43 +1,40 @@
 ﻿//Please read README section on Styling before editing or creating SCSS
 // This is your home for all shared themes and styles, please import it
 
-
 //Size Reference
 $scale: 1vw;
 
 /* Site colours */
-$background-color-dark: rgb(23, 3, 29);
-$background-color-light: rgb(182, 188, 244);
-$accent-color: rgb(121, 78, 255);
-$secondary-color-mid-tone: rgb(255, 14, 54);
-$foreground-color-light: rgb(244, 82, 79);
-$light-font: rgb(243, 215, 212);
-$image-background-color:rgb(132, 43, 91);
- 
-
-/* Rover Theme Colors */
-$curiosity-theme-color: rgb(161, 254, 161);
-$spirit-theme-color: rgb(255, 166, 188);
-$opportunity-theme-color: rgb(255, 235, 107);
+$curiosity-theme-color: rgba(94, 7, 30, 0.91);
+$theme-color-dark: rgb(12, 2, 16);
+$opportunity-theme-color:	rgba(206, 66, 87, 1);
+$spirit-theme-color:	rgba(255, 127, 81, 1);
+$theme-color-light: rgba(4, 1, 6, 0.55);
+$font-color-light: rgb(255, 255, 255);
 
 /* Site Fonts */
-$font-family-OpenSans:'Open Sans', sans-serif;
-$font-family-Ubuntu:'Ubuntu', sans-serif;
+$font-family-OpenSans:'Roboto', sans-serif;
+$font-family-headers:'Oxanium', cursive;
+
 
-$para-font-size:0.5em;
+$para-font-size:1em;
 $quote-font-size:.95em;
 $label-font-size:.90em;
-$navbar-font-size: 6*$scale;
-$navbar-font-size-larger-screens: 4*$scale;
+$navbar-font-size: 14pt;
+$navbar-font-size-larger-screens: 30pt;
```

Comment:
this is a nice change, but could we use `px` instead of `pt`?

pt is a slightly odd unit which tries to be a given 'length' on the screen. As screens change sizes a lot, px tends to be what you want a lot more often

Reply:
Hi Mike,
I'm actually going over all font sizing now to make sure it scales etc, can we merge this for now and I'll submit another pull request with font updates?

Reply:
yeah ok.